care/emr/api/viewsets/report/report_upload.py

```
@@ -0,0 +1,336 @@
+        instance.save()
+
+        data = self.pydantic_retrieve_model.serialize(instance, request.user).to_json()
+        return Response(data)
+
+    @extend_schema(
+        description="Get download URL for the report",
+        responses={200: "Download URL generated successfully"},
+        tags=["report"],
+    )
+    @action(detail=True, methods=["GET"])
+    def download(self, request, *args, **kwargs):
```

Ruff (0.14.6)

Unused method argument: `request`

(ARG002)